### DISCUSSION TITLE
Típusvizsgálat

### DISCUSSION THREAD

**Discussion by wlln22** [nav-gov-hu/eRECEIPT#34] [Kérdések és válaszok]
1. Ha egy felhőalapú epg több platformon is engedélyezett, akkor minden engedélyeztetett platform egy-egy hardver példányából meg kell őrizni egy etalon példányt? Eleget lehet tenni ennek az elvárásnak úgy, hogy egy hardveren több etalon példány is telepítve legyen?
Valamint, a tervezet szerint a "platform" hogyan értelmezendő, mit jelent pontosan?

2. _Sikeres teszthez a tesztrendszeren, tesztkörülmények között az e-nyugta kibocsátása és a vevői alkalmazásban való megjelenése között eltelt idő legfeljebb 5000 ms lehet._ Értelmezésünk szerint ez azt jelenti, hogy az 5000 ms teljesüléséhez 3 fél együttműködése, közös teljesítménye szükséges: az epg, a Nyugtatár és a vevői alkalmazás gyártójáé. A NAV által vállalt válaszidő benne van a fejlesztői dokumentációban. Amennyiben az epg és a vevői alkalmazás gyártója nem azonos, kinek a felelőssége, hogy az elvárt idő alatt megérkezzen a nyugta?

3. _"Titkosító kulcs fogadása az engedélyezési dokumentációban megjelölt POS terminálról és vevői alkalmazástól a kötelezően előírt és a dokumentációban megjelölt összes további módon."_ POS - bankkártya terminálnak kell képesnek lennie titkosító kulcs fogadására?

**Comment by BatariZ** [comment]
Az időközben megjelent rendelet (A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről), illetve az aktualizált Fejlesztői dokumentáció tartalmazza a válaszokat a feltett kérdésekre.
